kytos/core/api_server.py

```diff
@@ -118,6 +124,29 @@ def web_ui(self):
         """Serve the index.html page for the admin-ui."""
         return send_from_directory(self.flask_dir, 'index.html')
 
+    def update_web_ui(self, force=True):
+        """Update the static files for the Web UI.
+
+        Download the latest files from the UI github repository and update them
+        in the ui folder.
+        The repository link is currently hardcoded here.
+        """
+        repository = "https://github.com/kytos/ui"
+        uri = repository + "/blob/master/build/latest.zip?raw=true"
+
+        if not os.path.exists(self.flask_dir) or force:
+            # download from github
+            package = urlretrieve(uri)[0]
+            # remove files from web-ui/*
+            shutil.rmtree(self.flask_dir, ignore_errors=True)
+            os.mkdir(self.flask_dir)
+            # unzip and extract files to web-ui/*
+            zip_ref = zipfile.ZipFile(package, 'r')
+            zip_ref.extractall(self.flask_dir)
+            zip_ref.close()
```

Comment:
Here if the file is corrupted or something goes wrong, we lost the directory.

We don't need to use github api for now, but at least:

1. download zip file.
2. try unzip in a temp folder, for instance: `kytos-ui.latest`
3. move old folder to `kytos-ui.20171231`
4. move temp folder to kytos-ui